core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -226,6 +226,9 @@ require(
   'exploration-param-changes.service.ts');
 require(
   'pages/exploration-editor-page/services/exploration-param-specs.service.ts');
+require(
+  'pages/exploration-editor-page/services/exploration-image-counter.' +
```

Comment:
Make sure to import this in sorted order.

Reply:
done